content/learn/maintained.adoc

```diff
@@ -0,0 +1,65 @@
+---
+menu:
+  learn:
+    parent: Workflow
+title: Validated Patterns - Maintained Tier
+weight: 44
+aliases: /requirements/maintained/
+aliases: /requirements/validated/
+---
+
+:toc:
+
+= The Validated Patterns 'Maintained' Tier 
+
+[id="tldr"]
+== tl;dr
+
+The Maintained tier is intended to provide consumers with additional "sales" collateral and reassurance that the pattern was known to be functional on all currently supported LTS versions of OpenShift.  Inclusion in this tier may require additional work for the pattern's owner - which might be a partner or sufficiently motivated SME. 
```

Comment:
I'd avoid using the LTS expression since Red Hat almost never uses that. Did you really meant LTS as in EUS or as in whatever OCP is supported?

Reply:
I meant EUS